### ISSUE TITLE
Can it be used in elementary?

### ISSUE THREAD

**Issue by Yakumo-Yukari** [paysonwallach/appmenu#1]

**Comment by paysonwallach** [comment]
Yes, though full disclosure, I haven't tried to compile it let alone test it on elementary OS, but I don't see any reason why it couldn't.

**Comment by Yakumo-Yukari** [comment]
> Yes, though full disclosure, I haven't tried to compile it let alone test it on elementary OS, but I don't see any reason why it couldn't.

I installed it on elementary5 according to the documentation and it didn't work

**Comment by paysonwallach** [comment]
With the modified version of Wingpanel running, could you give me the output of the following command?

```
dbus-send --session --dest=org.freedesktop.DBus --type=method_call --print-reply /org/freedesktop/DBus org.freedesktop.DBus.ListNames | grep Registrar
```
